indico/modules/events/registration/models/forms.py

```diff
@@ -61,6 +63,11 @@ class RegistrationForm(db.Model):
         db.String,
         nullable=False
     )
+    is_participation = db.Column(
+        db.Boolean,
+        nullable=True,
```

Comment:
Simply create the column with `server_default='false'` in the migration script and remove the default right afterwards. No need to set it as nullable here.